.github/workflows/build-with-bratiska-cli-inhouse.yml

```diff
@@ -69,26 +110,8 @@ jobs:
       - name: Print Bratiska-cli version
         run: bratiska-cli deploy --version
 
-      - name: Creating kustomize file with Bratiska-cli
-        run: bratiska-cli build_kustomize --env=staging ${{ inputs.flag }} ${{ inputs.debug }} --no_image_repo_check --image=image_url_placeholder
-        continue-on-error: false
-
-      - name: Print pipeline summary - kustomize create
-        run: |
-          echo "###  Build summary :build:" >> $GITHUB_STEP_SUMMARY
-          echo ":partying_face: Bratiska-cli successfully created an kustomize file." >> $GITHUB_STEP_SUMMARY
-
-      - name: Print pipeline comment - kustomize build
-        if: github.event_name == 'pull_request'
-        uses: marocchino/sticky-pull-request-comment@v2.9.0
-        with:
-          header: Build Conditions
-          append: true
-          message: |
-            :partying_face: Bratiska-cli successfully created an kustomize file.
-
       - name: Running build with Bratiska-cli
-        run: bratiska-cli build_image --env=staging --build_image_no_registry ${{ inputs.flag }} --sentry=${{ secrets.sentry-token }} ${{ inputs.debug }}
+        run: bratiska-cli build_image --env=staging ${{ inputs.build_image_no_registry }} ${{ inputs.tag }} ${{ inputs.flag }} --sentry=${{ secrets.sentry-token }} ${{ inputs.debug }}
```

Comment:
I would expect for this to be just a simple replace with the above mentions steps. What sort of edge-cases are you expecting to have?

Reply:
1. building through `bratiska-cli`, will test if the docker image is able to build. It will literally run [`docker buildx build`](https://github.com/bratislava/bratiska-cli/blob/cb25bbc1770280b9b5f41c26e1ef722cf1de0123/src/commands.ts#L297). The GitHub action will do the same. Test if the image is able to build, but it also will have a benefit of not being maintained by us and will has a lot more options that we can freely specify, without change code or rebuilding another tool.
2. We are really far away from using `flux` or `argo`. There is nothing I would rather do then to just switch to either, but there needs to be a lot more work done before. Primarily on the cluster side or else we are risking that we will just do a bigger mess with additional complexity of other tooling.
3. if I'm not mistaken `bratiska-cli` will just ["rename" (copy/move)](https://github.com/bratislava/bratiska-cli/blob/cb25bbc1770280b9b5f41c26e1ef722cf1de0123/src/build_image/check_bratiska_build_envs.ts#L30C6-L30C18) those `.env.bratiska-cil-build.*` variable files. I think that can be achieved by simple `mv` command.

I will resolve this as we are short on time to delivery, but will definitely revisit this.